src/components/AppSidebar/AppSidebar.vue

```diff
@@ -23,6 +23,28 @@
 <!-- Follows the tab aria guidelines
 	https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html -->
 
+<docs>
+
+	### General description
+
+	This component provides a way to include the standardised sidebar.
+	The standard properties like title, subtitle, starred, etc. allow to automatically
+	include a standard-header like it's used by the files app.
+
+	### Usage
+
+	```
+	<AppSidebar
+		:title="title"
```

Comment:
Docs needs to work. Here title and subtitle does not exists as variables :)

Also could you also provide example for the regular one since we're at it? :p
We did not had the documentation back then! :grin: